features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -2790,6 +2790,23 @@
             <Scopes>internal_config_mgt_delete</Scopes>
         </Resource>
 
+        <Resource context="(.*)/api/server/v1/certificate-validation/(.*)" secured="true" http-method="POST">
```

Comment:
This is not needed

Reply:
fixed with https://github.com/wso2/carbon-identity-framework/pull/6448/commits/f5ee93c59a2b06cf45fd9151d5b3deab0bdf59c1